### PR TITLE
payment model, invoice model, registration queryset

### PR DESCRIPTION
**Pull request by connorfrendt**

### CODE REVIEW

File: hifireg/registration/models/payments.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from django.db import models
+ 
+class Payments(models.Model):
```
Comment (blueridger) [inline hifireg/registration/models/payments.py]: Need to delete this class, you already have "Payment" class. looks like a mishap from the git stuff we did.

**Review by blueridger — CHANGES_REQUESTED** [review]
1 change needed. Also it would be good to add to the PR description that the tests pass (if they do)

**Comment by connorfrendt** [comment]
Deleted the payments.py file (as there is already a payment.py file).
All of my tests pass as well!

**Comment by connorfrendt** [comment]
Deleted payments.py as requested, and all of my tests pass as well!

**Review by blueridger — APPROVED** [review]

**Pull request merged**